pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -218,6 +220,23 @@ func (m *MigCluster) GetClusterSubdomain(c k8sclient.Client) (string, error) {
 	return clusterSubdomain, nil
 }
 
+// GetOperatorVersion retrieves the operator version from the respective controllers ConfigMap
+func (m *MigCluster) GetOperatorVersion(c k8sclient.Client) (string, string, string, error) {
+	clusterConfig, err := m.GetClusterConfigMap(c)
+	if err != nil {
+		return "", "", "", liberr.Wrap(err)
+	}
+	configMapNamespace := clusterConfig.Namespace
+	configMapName := clusterConfig.Name
+
+	operatorVersion, ok := clusterConfig.Data[OperatorVersionKey]
+	if !ok {
+		return "", configMapNamespace, configMapName, liberr.Wrap(err)
```

Comment:
I think we could avoid passing so many values back, because the source of truth for the `configmapNamespace` and `configMapName` are defined in consts `ClusterConfigMapName`, `VeleroNamespace`